templates/book/book_template.html

```diff
@@ -1,230 +1,184 @@
-        {% endif %}
-        <div class="title_sponsored">
-          {% if e.sponsored == "silver" or e.sponsored == "gold" %}
-          <u> {{ e.title }} </u>
-          {% else %}
-          {{ e.title }}
-          {% endif %}
-        </div>
-        {% else %}
-        <div class="title">
-          {{ e.title }}
-        </div>
-        {% endif %}
-        <div>
-          {{ d.session_name }} {{ e.startTime }} | {{ e.duration }}min
-        </div>
-        <div class="lecturer">
-          <em>
-            {{ e.lecturer }}
-            {% if e.coauthor %}
-            i {{ e.coauthor }}
-            {% endif %}
-            {% if e.sponsored %}
-            z firmy <b> {{ e.organization }} </b>
-            {% endif %}
-          </em>
-        </div>
-        <div class="body">
-          {% for p in e.abstract %}
-          <p> {{ p }} </p>
-          {% endfor %}
-        </div>
+        {% endif %} {% endfor %}
       </div>
-      {% endif %}
       {% endfor %}
     </div>
-    {% endfor %}
-  </div>
-
-  <div class="fun_pages locations-page">
-
-    <h2 class="places-title"> Pobliskie miejsca warte odwiedzenia </h2>
 
-    <ul class="locations attractions">
+    <div class="fun_pages locations-page">
+      <h2 class="places-title">Pobliskie miejsca warte odwiedzenia</h2>
 
-      {% for a in places.attractions %}
-      <li class="location attraction">
-        <div class="row flex-split fw-400">
-          <span>{{ a.name }}</span>
-          <span>{{ a.distance }} / {{ a.time }}</span>
-        </div>
-        <div class="row fw-200 grayout">
-          <span>{{ a.address }}</span>
-        </div>
-        <div class="row">
-          <span>
-            {{a.description}}
-          </span>
-        </div>
-        {% if a.pricing %}
-        <div class="row">
-          <span>
-            cennik: {{ a.pricing | join(', ') }}
-          </span>
-        </div>
-        {% endif %}
-      </li>
-      {% endfor %}
-    </ul>
+      <ul class="locations attractions">
+        {% for a in place.attractions %}
+        <li class="location attraction">
+          <div class="row flex-split fw-400">
+            <span>{{ a.name }}</span>
+            <span>{{ a.distance }} / {{ a.time }}</span>
+          </div>
+          <div class="row fw-200 grayout">
+            <span>{{ a.address }}</span>
+          </div>
+          <div class="row">
+            <span> {{a.description}} </span>
+          </div>
+          {% if a.pricing %}
+          <div class="row">
+            <span> <b>Cennik:</b> {{ a.pricing | join(', ') }} </span>
+          </div>
+          {% endif %}
+        </li>
+        {% endfor %}
+      </ul>
 
-    {% if places.transport %}
-    <h3 class="places-subtitle"> Transport publiczny </h3>
-    <div class="locations transport">
-      <p>
-        {{ places.transport }}
-      </p>
+      {% if place.transport %}
+      <h3 class="places-subtitle">Transport publiczny</h3>
```

Comment:
Lots of places we've ever been to don't actually have a good public transport access, so maybe we could remove this section?

Reply:
Even for Przesieka we have a few sentences describing how to get there, so I don't know. We can remove it but is it that big of a deal?